ref/general/about-open-liberty.adoc

```diff
@@ -0,0 +1,96 @@
+
+You can freely mix MicroProfile, Jakarta EE, and other features, according to the needs of your applications. Open liberty maintains regular 4-weekly releases, so new versions of features are available quickly. For more information, see link:/docs/ref/general/#microprofile.html[MicroProfile simplifies developing cloud-native Java microservices].
+
+=== Deploy, test and debug in development mode
+Run your server in development mode, or link://blog/2019/10/22/liberty-dev-mode.html[dev mode], for automatic reload and deployment, on-demand testing, and debugger support.
+With dev mode, Open Liberty automatically detects changes in your code, compiles them, and deploys them to your running server.
+You can run tests on demand, or even automatically, so that you get immediate feedback on your changes.
+
+=== Develop with zero migration
+With Open Liberty zero-migration architecture, you focus on writing your applications, not the APIs changing under you.
+Zero-migration architecture means that you can use existing, unmodified configuration and application files with an updated version of Open Liberty, without unexpected changes in behavior.
+Open Liberty offers full compatibility between product versions so that your existing APIs and behaviors are supported in each new release. For more information, see link:/guides/getting-started.html[Packaging and deploying applications].
+
+
+== Deploy in containers to any cloud
+
+Open Liberty can be containerized and deployed to any Kubernetes cloud. Check out these guides to learn how you can easily deploy to link:/guides/okd.html[OKD], link:/guides/cloud-openshift.html[OpenShift], link:/guides/cloud-azure.html[Azure Kubernetes Service], link:/guides/cloud-aws.html[Amazon Web Services], link:/guides/cloud-ibm.html[IBM Kubernetes Service], and more.
+
+=== Protect applications and servers
+Open Liberty supports application security with a robust set of security features.
+During development and testing, Open Liberty's built-in link:/docs/ref/general/#basic-registry.html[basic user registry and the QuickStart security element] offer easily configurable security frameworks to test access to your application in a lightweight, self-contained manner.
+The `securityUtility` supports plain text encryption, so you can easily secure user passwords with hash encoding.
+
+After deployment, the link:/docs/ref/feature/#appSecurity.html[Application Security] feature provides the core capability for the Jakarta EE Security API, while the link:/docs/ref/feature/#transportSecurity.html[Transport Security] feature enables secure TLS connections by using HTTPS.
+The link:/docs/ref/feature/#ldapRegistry.html[LDAP User Registry] feature supports authentication and authorization from external LDAP registries.
```

Comment:
"The link:/docs/ref/feature/#ldapRegistry.html[LDAP User Registry] feature supports authentication and authorization from external LDAP registries."
In this sentence, instead of "authentication and authorization from external LDAP registries." should it be "authentication and authorization with external LDAP registries.", or "authentication and authorization to external LDAP registries." ?